SS_readcontrol_330.tpl

```diff
@@ -5243,12 +5244,11 @@
         }
         else if(Selex_Std_AL==3)
         {
-            N_warn++; cout<<" EXIT - see warning "<<endl;  warning<<N_warn<<" "<<" stderr on combined length-age selectivity is under repair "<<endl; exit(1);
           if(Selex_Std_Pick(i)>nages)
           {
             N_warn++; cout<<" EXIT - see warning "<<endl;  warning<<N_warn<<" "<<" cannot select stdev for age bin greater than maxage "<<Selex_Std_Pick(i)<<" > "<<nages<<endl; exit(1);
           }
-          ParmLabel+="AgeLenSelex_std_"+NumLbl(Do_Selex_Std)+"_"+GenderLbl(g)+"_A_"+NumLbl0(age_vector(Selex_Std_Pick(i))+1)+CRLF(1);
+          ParmLabel+="AgeLenSelex_std_"+NumLbl(Do_Selex_Std)+"_GP1_"+GenderLbl(g)+"_A_"+NumLbl0(age_vector(Selex_Std_Pick(i))+1)+CRLF(1);
```

Comment:
Was this line intentional?

Reply:
Yes.  This makes it explicit that only GP1 gets this variance output.  Although such a change belongs on another branch.  No problem because they will both be merged at same time and have no conflict.

Reply:
ok. in the future, it would be good for unrelated changes like this to at least be in a separate commit, which I don't think happened here (on another branch may be unnecessary, like you mention).